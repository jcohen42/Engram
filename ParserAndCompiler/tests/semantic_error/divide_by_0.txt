Begin program:
	Set x equal to 8.
	Set y equal to 0.
	Set z equal to x divided by y.
	Output z.
End program.