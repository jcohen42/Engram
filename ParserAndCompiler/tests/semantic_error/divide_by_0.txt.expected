Error: Divide by 0. Exiting program.